docs/userGuide/syntax/popovers.md

```diff
@@ -74,29 +85,29 @@ This is the same <trigger for="pop:trigger_id">trigger</trigger> as last one.
 <include src="extra/triggers.md" />
 </panel><p/>
 
-****Options****
-
-Name | Type | Default | Description
----- | ---- | ------- | ------
-trigger | `String` |	`hover`	| How the Popover is triggered.<br>Supports: `click`, `focus`, `hover`.
-header{{slot_info_trigger}} | `String` | `''` | Popover header, supports MarkDown text.
-content{{slot_info_trigger}} | `String` | `''` | Popover content, supports MarkDown text.
-placement | `String` | `top` | How to position the Popover.<br>Supports: `top`, `left`, `right`, `bottom`.
+\***\*Options\*\***
 
+| Name                         | Type     | Default | Description                                                                                             |
+| ---------------------------- | -------- | ------- | ------------------------------------------------------------------------------------------------------- |
+| trigger                      | `String` | `hover` | How the Popover is triggered.<br>Supports: `click`, `focus`, `hover`.                                   |
+| header{{slot_info_trigger}}  | `String` | `''`    | Popover header, supports MarkDown text.                                                                 |
+| content{{slot_info_trigger}} | `String` | `''`    | Popover content, supports MarkDown text.                                                                |
+| src                          | `String` |         | The url to the remote page to be loaded as the content of the popover. Both .md and .html are accepted. |
```

Comment:
We can stylize the file extensions to use inline code like so: `.md` and `.html` are accepted.

Also, we can break the `src` description into multiple lines with `<br>`, just like `placement`